lib/portage/dbapi/__init__.py

```diff
@@ -76,7 +77,7 @@ def _cpv_sort_ascending(cpv_list):
             # dict to map strings back to their original values.
             cpv_list.sort(key=cmp_sort_key(dbapi._cmp_cpv))
 
-    def cpv_all(self):
+    def cpv_all(self) -> Sequence[str]:
```

Comment:
Similarly, this is always a list of str, never an actual str, but the type annotations allow the latter.

Reply:
Yes. It is implied by the docs that `Sequence` should not be used for return type annotations. `List`/`list` should be used instead.